sway/tree/root.c

```diff
@@ -390,3 +390,18 @@ void root_get_box(struct sway_root *root, struct wlr_box *box) {
 	box->width = root->width;
 	box->height = root->height;
 }
+
+void root_rename_pid_workspaces(const char *old_name, const char *new_name) {
+	if (!pid_workspaces.prev && !pid_workspaces.next) {
+		wl_list_init(&pid_workspaces);
+	}
+
+	struct pid_workspace *pw = NULL;
+
```

Comment:
nitpick: can you just remove the blank line here and have the declaration right about the `wl_list_for_each`?

Reply:
done